cartridges/int_cloudinary/cartridge/templates/default/include/cloudinaryGalleryJs.isml

```diff
@@ -1,5 +1,6 @@
-
 <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled && (pdict.cloudinary.galleryEnabled || pdict.cloudinary.isGalleryEnabled)}">
     <script src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL.replace('/all.js', '/image.js')}" fetchpriority="high"></script>
-    <script src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL}" onload="window.cldProductGallery = cloudinary; window.renderCloudinaryGalleryWidget(); window.renderCloudinarySetGalleryWidgets()"> </script>
+    <script src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL}" onload="window.cldProductGallery = cloudinary; window.renderCloudinaryGalleryWidget(); window.renderCloudinarySetGalleryWidgets()" fetchpriority="high"></script>
+<iselseif condition="${dw.system.Site.current.preferences.custom.CLDEnabled && dw.system.Site.current.preferences.custom.CLDGalleryEnabled}"/>
+    <script defer src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL}" onload="window.cldProductGallery = cloudinary; window.renderCloudinaryGalleryWidget(); window.renderCloudinarySetGalleryWidgets()"></script>
```

Comment:
@asad-rafter per testing, please add the below line - after the existing deferred call to CLDGalleryJSURL:

<script defer="defer" src="${dw.system.Site.current.preferences.custom.CLDGalleryJSURL.replace('/all.js', '/image.js')}"></script>